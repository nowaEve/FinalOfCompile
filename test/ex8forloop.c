int main(){
    int i=0;
    int sum = 0;
    for(i =0 ; i < 5 ;  i++){
        sum +=i;
    }
    print sum;
}