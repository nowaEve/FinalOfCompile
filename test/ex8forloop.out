24 19 0 5 25 13 0 0 12 13 0 1 1 0 0 12 13 0 0 12 15 -1 16 52 13 11 13 0 1 1 13 0 1 1 11 13 11 1 12 15 -2 13 11 13 13 11 0 1 1 12 15 -2 13 11 0 5 7 18 24 13 0 1 1 11 22 21 2